.github/scripts/svgo.config.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  plugins: [
+    {
+      name: 'preset-default'
+    },
+    {
+      name: 'removeDimensions',
+      params: {
+        overrides: {
+          removeDimensions: true,
+        }
+      }
+    },
+    {
+      name: 'convertStyleToAttrs',
```

Comment:
That's a lot of style. 😎😂
I think `inlineStyles` can group everything by itself, so we would only need to remove this plugin. However I'll be testing it first.
Then, if it works like that, I'll be removing this plugin in next commit, otherwise I'll be incorporating your idea to this set. 😁